guides/common/modules/con_automating-project-management-with-project-ansible-collections.adoc

```diff
@@ -0,0 +1,11 @@
+[id="automating-{project-context}-management-with-{project-context}-ansible-collections"]
+= Automating {Project} management with {Project} Ansible Collections
```

Comment:
```suggestion
= Automating {Project} management with {Project} Ansible Collection
```

It's really just the one we're describing here.
(applies to the whole PR)

(We have the operation collection too, but that is not documented and does other tasks)

Reply:
Updated.